BrytescoreObjCAPI/BrytescoreObjCAPI/Classes/BrytescoreAPIManager.m

```diff
@@ -13,14 +13,14 @@ @implementation BrytescoreAPIManager
 static NSString* packageName = @"/package.json";
 static NSString* hostname = @"com.brytecore.mobile";
 static NSString* library = @"iOS";
-static NSString* libraryVersion = @"0.1.2";
+static NSString* libraryVersion = @"1.3.0";
```

Comment:
@mcgomez since we're jumping versions, should we bump to 1.4.0 to sync with the standard iOS lib?